src/controllers/userController.js

```diff
@@ -0,0 +1,23 @@
+const models = require('../models');
+export const user = {
+  async createUser(req, res) {
+    try {
+      const createdAt = new Date();
+      const updatedAt = new Date();
+      const userData = {
+        email: req.body.email,
+        password: req.body.password,
+        username: req.body.username,
+        phoneNumber: req.body.phoneNumber,
+        updatedAt,
+        createdAt
+      }
+      const post = await models.User.create(userData);
+      return res.status(201).json({
+        post
```

Comment:
I don't think you should just send the user data just like that as json, you can still add other fields like status, message and data. the user data can be inside the data field